runtime/doc/diagnostic.txt

```diff
@@ -97,8 +97,8 @@ If a diagnostic handler is configured with a "severity" key then the list of
 diagnostics passed to that handler will be filtered using the value of that
 key (see example below).
 
-Nvim provides these handlers by default: "virtual_text", "signs", and
-"underline".
+Nvim provides these handlers by default: "virtual_text", "virtual_lines",
+"signs", and "underline".
```

Comment:
In `:help nvim_buf_set_extmark()` the related fields are named `virt_text`, `virt_lines`. Do we want to be consistent (though that would imply deprecating `virtual_text` here and aliasing it to `virt_text`)?

Reply:
Yeah. I find consistency with the existing handler's name preferable, and I personally don't find the hassle of renaming (and deprecating) `virtual_text` worth it. Letting others weigh in though.